internal/armador/chartList_test.go

```diff
@@ -206,24 +206,68 @@ func TestChartList_processCharts(t *testing.T) {
 					Dependencies: []Chart{Chart{Name: "first-dep", Repo: "github", PathToChart: "helm/first-dep"}},
 				},
 				"first-dep": Chart{Name: "first-dep", Repo: "github",
-					ChartPath: "tp/cache/charts/first-dep", Packaged: false,
+					ChartPath: "tp/cache/charts/first-dep/helm/first-dep", Packaged: false, PathToChart: "helm/first-dep",
 				},
 				"secondchart": Chart{Name: "secondchart", Repo: "stable", ChartPath: "../testData"},
 			},
 			wantErr: false,
 		},
+		{
+			name: "complex-dependecy",
+			charts: &ChartList{
+				"first-chart": Chart{Name: "first-chart", Repo: "stable", ChartPath: "../testData",
+					Dependencies: []Chart{Chart{Name: "first-dep", Repo: "github", Version: "1.0.0", Packaged: false, PathToChart: "helm/first-dep"}}, Packaged: false, PathToChart: "helm/first-chart",
```

Comment:
my first thought is what makes this test "complex", it's not much different than the previous test case - i do see enough difference to leave it in, but maybe change the name

also, the order/format of the values makes it hard to read what's going on - try to make it more clear for future people coming to the code